pre_commit_mirror_maker/languages.py

```diff
@@ -4,6 +4,7 @@
 import subprocess
 import urllib.request
 
+import lxml.html
```

Comment:
you should not need an external dependency to read xml -- there's an `xml` stdlib module